backend/packages/Upgrade/src/api/services/ExperimentAssignmentService.ts

```diff
@@ -856,37 +856,60 @@ export class ExperimentAssignmentService {
    * experiment - Experiment definition
    */
   private async checkEnrollmentEndingCriteriaForCount(experiment: Experiment): Promise<void> {
-    const { enrollmentCompleteCondition, group, partitions } = experiment;
+    const { enrollmentCompleteCondition } = experiment;
     const { groupCount, userCount } = enrollmentCompleteCondition;
 
     // get assignments and fetch monitored document for those assignments
-    const getMonitoredDocumentOfExperiment = async (experimentDoc: Experiment) => {
-      // get groupAssignment and individual assignment details
-      const individualAssignments = await this.individualAssignmentRepository.find({
-        where: { experiment: experimentDoc },
-        relations: ['user'],
-      });
+    const getMonitoredDocumentOfExperimentTypeGroup = async (experimentDoc: Experiment) => {
 
-      // get the monitored document for all the partitions in the experiment
-      const experimentPartitionIds = partitions.map((partition) => {
-        const experimentId = partition.expId;
-        const experimentPoint = partition.expPoint;
-        return getExperimentPartitionID(experimentPoint, experimentId);
-      });
+      const partitionId = experimentDoc.partitions.map((p) => {
+        return `${p.expId}_${p.expPoint}`
```

Comment:
We have a utility function `getExperimentPartitionID` inside `ExperimentPartition.ts`